animation_data.cpp

```diff
@@ -20,25 +25,76 @@ std::vector<SDL_Rect> BuildAnimationRects(SDL_Rect start,
 
   return result;
 }
+
+// Takes the number of frames and sets the offset for each frame of the
+// animation to zero.
+std::vector<Location> NoOffsets(int number_of_frames) {
+  std::vector<Location> result;
+  for (int i = 0; i < number_of_frames; i++) {
+    result.push_back({0, 0});
+  }
+
+  return result;
+}
 }  // namespace
 
 AnimationMap GetGokuAnimations() {
   return {{AnimationType::kDefault,
            {.type = AnimationType::kDefault,
             .time_between_frames_ms = 300,
+            .offsets = NoOffsets(/*number_of_frames*/ 2),
             .image_rects =
                 BuildAnimationRects(kGokuIdleStart, /*number_of_frames=*/2),
             .priority = 0}},
           {AnimationType::kRunRight,
            {.type = AnimationType::kRunRight,
             .time_between_frames_ms = 100,
+            .offsets = NoOffsets(/*number_of_frames*/ 2),
             .image_rects =
```

Comment:
let's rename this to animation_frames (or whatever you decide to call the struct)